1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -326,13 +326,15 @@ def create_infofiles(row, filenames_file, not_ok_file):
             personnamn_not_even = True
             infotext += "|depicted people     = " + row["Personnamn / avbildad"] + "\n"
         else:
+            if "Linné, Sigvald" in row["Personnamn / avbildad"]:
+                linne_category = True
+                depicted_people += "[[:d:Q5959424|Sigvald Linné]]\n"
             for i, j in zip(lista[::2], lista[1::2]):
-                if j + " " + i == "Sigvald Linne":
-                    linne_category = True
-                    depicted_people += "[[:d:Q5959424|Sigvald Linné]]\n"
-                    content_categories_string += "[[Category:Sigvald_Linné]]\n"
+                if j + " " + i == "Sigvald Linné":
+                    depicted_people += "[[Category:Sigvald_Linné]]"
                 else:
-                    depicted_people += j + " " + i + "/"
```

Comment:
Fixed the / in 267861c34fe1161c475f962e62d2c6d0d539a91f

Do you also really mean I should delete lines 331, 332, 333?